mii/client.py

```diff
@@ -12,17 +12,27 @@
 from mii.method_table import GRPC_METHOD_TABLE
 
 
-def _get_deployment_info(deployment_name):
-    configs = mii.utils.import_score_file(deployment_name).configs
-    task = configs[mii.constants.TASK_NAME_KEY]
-    mii_configs_dict = configs[mii.constants.MII_CONFIGS_KEY]
+def _get_deployment_info(deployment_tag):
+    deployments = []
```

Comment:
We could rename this to `deployment_configs` to be more accurate. We don't want to get confused about what this list holds.

Reply:
This change could probably be propagated to the rest of the code too (unless I'm misunderstanding what is held in this variable)